Source/WebCore/page/scrolling/ScrollingTreeGestureState.h

```diff
@@ -29,6 +29,7 @@
 
 #include <WebCore/ScrollTypes.h>
 #include <wtf/Markable.h>
+#include <wtf/WeakPtr.h>
```

Comment:
Wrong include for `ThreadSafeWeakPtr`. Should include `<wtf/ThreadSafeWeakPtr.h>`